app/code/Magento/GoogleAnalytics/Helper/Data.php

```diff
@@ -23,6 +23,8 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
 
     const XML_PATH_ACCOUNT = 'google/analytics/account';
 
+    const XML_PATH_OPTIMIZE_CONTAINER_ID = 'google/analytics/optimize_container_id';
```

Comment:
@ihor-sviziev you made it breaking compatibility since the class is marked with `@api`

Reply:
Please consider moving it into another class